releasenotes/notes/generate_pass_manager_preset-1e6c9641accd5d60.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Added a new function :func:`qiskit.transpiler.generate_preset_pass_manager` which can
```

Comment:
Should we also add a reno for `target_to_backend_properties` which is now public?

Reply:
I really didn't want to advertise it too much. It's public because I'm using across multiple modules and it felt weird to do otherwise but I'm trying to move everything away from the `BackendProperties` object. 

As for the test coverage we have implicit coverage already via tests that use `transpile()`. We've had test coverage for that since #7227 I can add dedicated tests for though if you'd prefer to have that since it is a standalone function now.

Reply:
No that's fine with me, it's also not included in any `__init__` 👍🏻